collectionfunctions.php

```diff
@@ -81,7 +93,49 @@ function checkNew(PDO $database) {
                 header('Location: collectionchallenge.php');
```

Comment:
Might be missing something but can this go in the if statement above? :)

Reply:
This is where I actually call the execute, but I need to do the checks in the if statement above before I do 😇